class/main.rb

```diff
@@ -1,19 +1,57 @@
 require_relative 'app'
 require_relative 'execute'
+require 'json'
+
 class Main
   def display_ui
     puts 'Please choose an option by entering a number:'
+    load_data # Moved this line here
     puts '1 - List all books'
     puts '2 - List all people'
     puts '3 - Create a person'
     puts '4 - Create a book'
     puts '5 - Create a rental'
     puts '6 - List all rentals for a given person id'
-    puts '7 - Exit'
+    puts '7 - Save data And Exit'
     user_choice = gets.chomp.to_i
     puts user_choice
     Execute.new(user_choice).execute
   end
+
+  # rubocop:disable Metrics/MethodLength
+  def load_data
+    books = JSON.parse(File.read('books.json'))
```

Comment:
- [ ] There is an issue with your app. When starting it, it fails to load the books JSON file (though it exists) and crashes. Kindly fix this issue. Note that you should handle situations where the JSON files exist or not.

  ![image](https://github.com/HtetWaiYan7191/school-library/assets/54848363/be862622-97e7-4aa6-92cf-1d6983a3eea5)

Reply:
@DammyShittu  
# Hello, please check the screenshot below: 
![image](https://github.com/HtetWaiYan7191/school-library/assets/15729467/935433a0-22bd-4de7-b299-e06c1716e8ac)

Reply:
- Good job done so far, but I am also experiencing the same issues as the previous reviewer, a reason for this is that you are loading your resources i.e books, person, and rentals from the wrong path. Please note that your `.json` files are in the `class` directory. Please fix this 🙏 
**OPTIONAL** - Also, when I  delete all `.json` and load the resources afresh, I get the error below. I suggest you include a condition in your code to initialize an empty array when there are no `.json` files. Please fix this 🙏 
![rgqerg](https://github.com/HtetWaiYan7191/school-library/assets/73607512/b6c789d2-11b9-47b8-94f3-a825fa19696c)
**OPTIONAL** - Your load data method loads all resources when the app is initialized, this is not the expected behavior for the application. To enhance user experience, I suggest you only load a resource according to user input/options. Please refactor your load data method to ensure each resource has its own method when they are called. Please fix this 🙏 
![42424](https://github.com/HtetWaiYan7191/school-library/assets/73607512/b8812a57-d3ef-4dbc-af4d-8ade5c29d054)